worlds/tits_the_3rd/client/memory_io.py

```diff
@@ -148,15 +149,15 @@ def _read_int(self, offset, byteorder: Literal["little", "big"] = "big"):
         data = self._read_bytes(offset, 4)
         return int.from_bytes(data, byteorder)
 
-    def _read_short(self, offset, byteorder: Literal["little", "big"] = "big"):
+    def _read_short(self, offset, byteorder: Literal["little", "big"] = "big", signed: bool = False):
```

Comment:
Nit: if we're going to add a signed arg here we should also add it to the other read functions.